parsec/core/fs/storage/workspace_storage.py

```diff
@@ -319,3 +358,6 @@ async def set_manifest(
 
     async def ensure_manifest_persistent(self, entry_id: EntryID) -> None:
         pass
+
+    def to_timestamped(self, timestamp: Pendulum) -> "WorkspaceStorageTimestamped":
+        return WorkspaceStorageTimestamped(self, timestamp)
```

Comment:
return typing https://docs.python.org/3/library/typing.html#typing.NoReturn ?

Reply:
I updated methods that call `_throw_permission_error` with `NoReturn`, but this one (`to_timestamped`) returns something